Examples/Examples-iOS/IGListKitExamples/ViewControllers/MixedDataViewController.swift

```diff
@@ -23,7 +23,8 @@ final class MixedDataViewController: UIViewController, ListAdapterDataSource {
     let collectionView = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout())
 
     let data: [Any] = [
```

Comment:
@weyert what about changing this to

```swift
let data: [ListDiffable]
```

You might have to soft-cast the stuff in the array literal, like

```swift
"String" as ListDiffable,
// etc
```

But its not a force-cast. Wanna give that a shot? Will get rid of the fatals too I think.